dashboards/main/metrictank.json

```diff
@@ -3659,16 +3666,41 @@
         {
           "refId": "A",
           "target": "alias(sumSeries(metrictank.stats.$environment.$instance.cache.size.max.gauge64), 'max')",
-          "textEditor": false
+          "textEditor": false,
+          "refCount": 1
         },
         {
           "refId": "B",
-          "target": "alias(sumSeries(metrictank.stats.$environment.$instance.cache.size.used.gauge64), 'used')"
+          "target": "alias(sumSeries(metrictank.stats.$environment.$instance.cache.size.used.gauge64), 'used')",
+          "refCount": 1
         },
         {
           "refId": "C",
           "target": "alias(divideSeries(sumSeries(metrictank.stats.$environment.$instance.cache.size.used.gauge64),sumSeries(metrictank.stats.$environment.$instance.cache.size.max.gauge64)), 'utilisation')",
-          "textEditor": true
+          "textEditor": true,
+          "refCount": 1
+        },
+        {
+          "refId": "D",
+          "target": "alias(sumSeries(metrictank.stats.$environment.$instance.cache.overhead.chunk.gauge64), 'chunk overhead')",
+          "refCount": 1
+        },
+        {
+          "refId": "E",
+          "target": "alias(sumSeries(metrictank.stats.$environment.$instance.cache.overhead.flat.gauge64), 'flat overhead')",
+          "refCount": 1
+        },
+        {
+          "refId": "F",
+          "target": "alias(sumSeries(metrictank.stats.$environment.$instance.cache.overhead.lru.gauge64), 'lru overhead')",
+          "refCount": 1
```

Comment:
you can combine these 3 into one query and use `aliasByNode()` to set the right alias for all of them